mlos_core/MANIFEST.in

```diff
@@ -1,3 +1,5 @@
 # setuptools-scm has an issue that causes it to include all files in git in the sdist produced.
 # Here we exclude test files.
 prune **/tests
+# Also exclude the example notebooks.
+prune notebooks/
```

Comment:
Exclude some files that weren't supposed to be in the whl or sdist (also in the wrong dir).